docs/03-Usage.md

```diff
@@ -114,7 +113,7 @@ Similar to the FulltextSearch module, using init supports all basic methods to a
 Available methods are:
 
 | Method | Purpose | Required | Usage |
-|-|-|-|-|
+| ------ | ------- | -------- | ----- |
```

Comment:
Cheers, I've been wanting to make the markdown a tad more readable, especially tables.